test/base/EtherCollateralLoansTakeOutLoanTest.js

```diff
@@ -148,7 +148,7 @@ contract('EtherCollateralLoansTakeOutLoanTest', function (accounts) {
                 assert(tx);
 
                 const txTime = (await web3.eth.getBlock(tx.receipt.blockNumber)).timestamp
-                const interestOwed = Math.floor(amountToBorrow * 1475 * loanDuration / 10000 / 3650000)
+                const interestOwed = Math.floor(amountToBorrow * 1475 * loanDuration / 10000 / 31536000)
```

Comment:
Maybe keep this number as a constant somewhere instead of it being a magic number 😅